web/libs/editor/src/common/Hotkey/WithHotkey.tsx

```diff
@@ -0,0 +1,36 @@
+import { Children, cloneElement, forwardRef, type ReactElement } from "react";
+import { Hotkey } from "../../core/Hotkey";
+import type { ButtonProps } from "@humansignal/ui";
+import type Keymaster from "keymaster";
+import { useHotkey } from "../../hooks/useHotkey";
+import { isDefined } from "../../utils/utilities";
+
+type HotkeyProps = {
+  binging?: keyof typeof Hotkey.keymap;
+  hotkeyScope?: string;
+  displayedHotkey?: keyof typeof Hotkey.keymap;
+  children: ReactElement<ButtonProps>;
+};
+export const WithHotkey = forwardRef<HTMLElement, HotkeyProps>(
+  ({ children, binging: hotkey, hotkeyScope, displayedHotkey }: HotkeyProps, ref) => {
+    Children.only(children);
+
+    const { onClick, tooltip } = children.props;
+    const clone = cloneElement(children, { ref, tooltip: undefined, tooltipTheme: "dark" });
```

Comment:
should we be defining the tooltip theme? how does this interact with dark-mode? cc @bmartel @ricardoantoniocm